src/app/tabs/runTab/contractDropdown.js

```diff
@@ -51,10 +51,11 @@ class ContractDropdownUI {
     this.atAddressButtonInput = yo`<input class="${css.input} ${css.ataddressinput} ataddressinput form-control" placeholder="Load contract from Address" title="address of contract" oninput=${this.atAddressChanged.bind(this)} />`
     this.selectContractNames = yo`<select class="${css.contractNames} custom-select" disabled></select>`
 
-    this.createPanel = yo`<div class="${css.button}"></div>`
+    this.createPanel = yo`<div class="${css.deployDropdown}"></div>`
     this.orLabel = yo`<div class="${css.orLabel}">or</div>`
     let el = yo`
-      <div class="${css.container}">
+      <div class="${css.container} module-section">
+        <div class="${css.settingsLabel} form-control-label">Contract</div>
```

Comment:
is form-control-label boostrap class?

Reply:
.form-control-label class is not bootstrap class but my custom class. Bootstrap styles label using tag selector. I added this because you didn't use <label> tag where it should be used. Sometime you used just <div> tag instead of <label> tag.

Reply:
When we have used a div tag instead of label - it was bad form on our side.  The tags need to updated to be <label> tags.